### PR TITLE
Misc: some code review

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping <hukeping@huawei.com>

### CODE REVIEW

**Comment by riyazdf** [comment]
jenkins, test this please.

File: tuf/data/targets.go [hunk 1]
```diff
@@ -107,7 +107,10 @@ func (t *SignedTargets) BuildDelegationRole(roleName string) (DelegationRole, er
 				pubKey, ok := t.Signed.Delegations.Keys[keyID]
 				if !ok {
 					// Couldn't retrieve all keys, so stop walking and return invalid role
-					return DelegationRole{}, ErrInvalidRole{Role: roleName, Reason: "delegation does not exist with all specified keys"}
+					return DelegationRole{}, ErrInvalidRole{
+						Role:   roleName,
+						Reason: "delegation does not exist with all specified keys, " + keyID + " missing",
```
Comment (cyli) [inline tuf/data/targets.go]: Thank you so much for adding more information!  I was wondering if we could clarify this by saying something like `"role lists unknown key " + keyID + " as a signing key"` or something?

**Comment by riyazdf** [comment]
thanks for this contribution @HuKeping!  LGTM after @cyli's comments

**Comment by HuKeping** [comment]
updated.

**Comment by riyazdf** [comment]
jenkins, test this please

**Comment by riyazdf** [comment]
jenkins, test this please.

**Pull request merged**